docker/Dockerfile

```diff
@@ -31,8 +31,10 @@
 FROM openjdk:11-slim
 
 # Set following arguments with compatible versions
-ARG harvest_version=3.5.0
-ARG reg_manager_version=4.2.0
+ARG harvest_version=3.6.0-SNAPSHOT
+#ARG harvest_version=3.5.0
```

Comment:
Do we want to keep these commented out lines?
```
#ARG harvest_version=3.5.0
#ARG reg_manager_version=4.2.0
```

Reply:
Ok, I remove them